vllm/model_executor/openvino_model_loader.py

```diff
@@ -449,7 +449,13 @@ def get_model(model_config: ModelConfig,
     if is_openvino_optimum_intel():
         import openvino as ov
         from optimum.intel import OVModelForCausalLM
-        pt_model = OVModelForCausalLM.from_pretrained(model_config.model, export=True, compile=False, load_in_8bit=False, trust_remote_code=True) # need stateful because it also enables SDPA
+        pt_model = OVModelForCausalLM.from_pretrained(
+            model_config.model,
+            export=True,
+            compile=False,
+            load_in_8bit=False,
```

Comment:
should we also remove `load_in_8bit=False` ?

Reply:
Do we have an external option for that? `quantization`?

Reply:
Indeed:

--quantization (-q) {awq,squeezellm,None}
Method used to quantize the weights.